### PR TITLE
Remove all DEVEL refs [DI-241]

### PR DESCRIPTION
**Pull request by nishaatr**
Remove all DEVEL refs as we don't package DEVEL - only produce Docker image 
See JIRA for more details

Not sure about the below as url is pointing to JFrog `devel` repo. **I think we can delete but do confirm?**

https://github.com/hazelcast/homebrew-hz/blob/master/hazelcast-enterprise%405.1.dr6.rb
https://github.com/hazelcast/homebrew-hz/blob/master/hazelcast-enterprise%405.1.dr8.rb

image

Accompanied by https://github.com/hazelcast/hazelcast-packaging/pull/224

Fixes [DI-241](https://hazelcast.atlassian.net/browse/DI-241)

[DI-241]: https://hazelcast.atlassian.net/browse/DI-241?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by JackPGreen — CHANGES_REQUESTED** [review]
Because these files are dynamically generated, I think any modification should wait until the generator (i.e. https://github.com/hazelcast/hazelcast-packaging/pull/224) is updated.
Otherwise an intermediate build will cause conflicts.

**Comment by nishaatr** [comment]
> Because these files are dynamically generated, I think any modification should wait until the generator (i.e. [hazelcast/hazelcast-packaging#224](https://github.com/hazelcast/hazelcast-packaging/pull/224)) is updated. Otherwise an intermediate build will cause conflicts.

Makes sense. thanks

**Review by ldziedziul — APPROVED** [review]
> Remove all DEVEL refs as we don't package DEVEL - only produce Docker image See JIRA for more details
> 
> Not sure about the below as url is pointing to JFrog `devel` repo. **I think we can delete but do confirm?**

DR means devel release, it was a previous name, we can delete them as well

**Comment by nishaatr** [comment]
> > Remove all DEVEL refs as we don't package DEVEL - only produce Docker image See JIRA for more details
> > Not sure about the below as url is pointing to JFrog `devel` repo. **I think we can delete but do confirm?**
> 
> DR means devel release, it was a previous name, we can delete them as well

Ah. should have been obvious. Normally DR means disaster recovery!
I will remove them soon. thanks

REMOVED: [2cdc9e7f4](https://github.com/hazelcast/homebrew-hz/pull/38/commits/2cdc9e7f45bedf2067165454daf71c4038c9fb10)

**Comment by ldziedziul** [comment]
It's good that you haven't removed anything you wasn't sure 💪

**Comment by nishaatr** [comment]
> It's good that you haven't removed anything you wasn't sure 💪

thanks. What can I say, I am risk averse. In my nature + I think I have learnt the hard way :slightly_smiling_face:

File: hazelcast-enterprise@5.1.dr6.rb
Comment (JackPGreen) [inline hazelcast-enterprise@5.1.dr6.rb:1]: You've deleted this file, but the other files still contain a `conflicts_with` entry referencing it, leading to a warning.

**Review by JackPGreen — CHANGES_REQUESTED** [review]

File: hazelcast-enterprise@5.1.dr6.rb
Comment (nishaatr) [reply hazelcast-enterprise@5.1.dr6.rb:1]: Opppss. Let me fix that . thanks

Comment (nishaatr) [reply hazelcast-enterprise@5.1.dr6.rb:1]: Fixed [5e3926](https://github.com/hazelcast/homebrew-hz/pull/38/commits/5e3926945cc13d07395cedd7ed117eafab5fee6d)

Good catch. thanks

**Review by JackPGreen — APPROVED** [review]

**Pull request merged**